openc3-cosmos-init/plugins/packages/openc3-tool-common/src/tools/base/components/Notifications.vue

```diff
@@ -273,6 +269,8 @@ export default {
   created: function () {
     this.showToast = localStorage.notoast === 'false'
     this.subscribe()
+    // TODO How does this get updated after initialization
+    this.alerts = this.$store.state.notifyHistory
```

Comment:
Alerts / notifications still needs some work. I think all the information is here. I left the current implementation where the bell icon always shows and everything is basically a notification.